test/cljs_time/coerce_test.cljs

```diff
@@ -81,7 +81,8 @@
   (is (= 893462400 (to-epoch (js/Date. 893462400000))))
   (is (= (long 0) (to-epoch 0)))
   (is (= 893462400 (to-epoch 893462400000)))
-  (is (= 893462400 (to-epoch "1998-04-25T00:00:00.000Z"))))
+  (is (= 893462400 (to-epoch "1998-04-25T00:00:00.000Z")))
+  (is (= 893462400 (to-epoch "1998-04-25T00:00:00.500Z"))))
```

Comment:
Might be good to add a comment or a `testing` block around these two tests to explain why they are there. It's quite easy to miss the `.500Z` difference in the second one.